src/export/digitalPlanning/mocks/planningPermission.ts

```diff
@@ -15,6 +15,18 @@ export const mockPlanningPermissionSession = {
   sanitised_at: null,
   email: "example@example.com",
   passport: {
+    "proposal.materials": [
+      {
+        type: "window",
+        existing: "Glaze window",
+        proposed: "Glaze window",
+      },
+      {
+        type: "wall",
+        existing: "Red bricks",
+        proposed: "Brown bricks",
+      },
+    ],
```

Comment:
This is a snippet direct from Meera's example. At some point we'll want to fully regenerate all these mocks, but piecemeal additions are working okay in the meantime. 

My process for this is:
- Add mock `passport` data values
- Temporarily edit `src/export/digitalPlanning/model.test.ts` to log out properties of interest
![Screenshot from 2025-03-05 13-28-25](https://github.com/user-attachments/assets/c23e88fd-9a22-45c4-8b93-b42b32f5579f)
- Run `pnpm test` & confirm logs are as expected 
![Screenshot from 2025-03-05 13-26-20](https://github.com/user-attachments/assets/5bee81cf-9bb8-49c9-a998-43d6dfa0b684)